cmake/install_helpers.cmake

```diff
@@ -8,27 +8,32 @@ set(GINKGO_INSTALL_PKGCONFIG_DIR "${CMAKE_INSTALL_LIBDIR}/pkgconfig")
 set(GINKGO_INSTALL_CONFIG_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/Ginkgo")
 set(GINKGO_INSTALL_MODULE_DIR "${CMAKE_INSTALL_LIBDIR}/cmake/Ginkgo/Modules")
 
-function(ginkgo_install_library name subdir)
-    # This setting sets the directory of the library as the `INSTALL_RPATH`.
-    # This is required for `libginkgo.so` to find its dependencies such as
-    # `libginkgo_omp.so` etc. Note that with this setting it's not possible on
-    # Linux to control which Ginkgo libraries are found and where: if all the
-    # libraries are present in the directory, these will be used first and
-    # foremost and the environment variable `LD_LIBRARY_PATH` will have no
-    # effect.
+function(ginkgo_add_install_rpath name subdir)
```

Comment:
I can't see the subdir parameter being used anywhere. Is it still relevant here?

Reply:
Right, it's a relica of `ginkgo_install_library` where it's also useless now. Since this has been annoying me for a while, I'm removing this parameter everywhere.